src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -280,6 +280,62 @@ InModuleScope Arcus.Scripting.LogicApps {
                 # Act
                 Disable-AzLogicAppsFromConfig -DeployFileName "$PSScriptRoot\Files\LogicApps\deploy-orderControl-noWaitingOrRunningRunsWithSingleImmediate.json" -ResourceGroupName $resourceGroup
 
+                # Assert
+                Assert-MockCalled Get-AzCachedAccessToken -Scope It -Times 1
+                Assert-MockCalled Disable-AzLogicApp -Scope It -Exactly 1 -ParameterFilter { $ResourceGroupName -eq $resourceGroup -and $LogicAppName -eq "snd-async" }
+            }
+            It "Disables all logic apps with option maximumFollowNextPageLink set to 25" {
```

Comment:
Good to have these tests as alternative to integration tests, as I'm guessing it is troublesome to set this up?